robots/lwr_robotiq_3finger.urdf.xacro

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="lwr">
+
+  <xacro:arg name="use_gazebo" default="1"/>
+  <xacro:arg name="use_transmission" default="1"/>
+  <xacro:arg name="collision_simple" default="1"/>
+  <xacro:arg name="visual_simple" default="0"/>
+
+  <!-- KUKA LWR4 arm -->
+  <xacro:include filename="$(find kuka_lwr4_description)/robots/kuka_lwr_arm.urdf.xml"/>
+
+  <!-- Robotiq 3 finger hand -->
+  <xacro:include filename="$(find robotiq_s_model_visualization)/cfg/s-model_articulated_macro.xacro" />
```

Comment:
Try to use the newer, ``kinetic-devel`` head here: the package is now renamed to be ``robotiq_3f_gripper_visualization``.

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic, I therefore assumed that it is not ready yet.
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Huskies?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic, I therefore assumed that it is not ready yet.
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Huskies?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Huskies?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Huskies?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Husky?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Husky?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Husky?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Husky?

Reply:
`jade-devel` is still the default branch and the `robotiq` package hasn't been released yet to kinetic. I therefore assumed that it is not ready yet (http://repositories.ros.org/status_page/ros_kinetic_default.html?q=robotiq).
Are you successfully using the newer `kinetic-devel` branch on the dual-arm Husky?